src/__test__/calculate.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  render, screen, fireEvent,
+} from '@testing-library/react';
+import Calculator from '../components/Calculator';
+import calculate from '../logic/calculate';
+
+describe('Test calculate function', () => {
+  const obj = {
+    total: null,
+    next: null,
+    operation: null,
+  };
+  it('Checking for complete operation', () => {
+    render(<Calculator />);
+    const one = screen.getByText('1');
+    const sum = screen.getByText('+');
+    const two = screen.getByText('2');
+    const equals = screen.getByText('=');
+    const display = screen.getByTestId('display');
+    fireEvent.click(one);
+    fireEvent.click(sum);
+    fireEvent.click(two);
+    fireEvent.click(equals);
+    expect(display.innerHTML).toBe('3');
+  });
+
+  it('Checking for complete operation', () => {
```

Comment:
- [Optional] Hello Tea, Good job on the test, But remember your `it(message)` block is expected to explain what the test is doing inside. as part of that every `it(message)` block should have a different message explaining what it assers.